ng2-material/components/pagination/README.md

```diff
@@ -0,0 +1,67 @@
+# MdPagination
```

Comment:
I'd like to start formatting the readme files consistently. I took some formatting from you and the material2 readmes, and the stuff in dialog component felt pretty nice: https://github.com/justindujardin/ng2-material/tree/master/src/components/dialog

I did a few things:

 - use the component selector as H2 `##`
 - have H3 `###` for `Properties`, `Events`, `Examples`
 - at least one example for each described selector in the page.
 - try to keep the examples as small as possible, showing off only the selector being described. (the more complete examples should live in the live component examples on the page.)

Can you check out what I did for `0.4.1` site and let me know how you like it? https://justindujardin.github.io/ng2-material/#/components/dialog